programs/autocrat/src/instructions/dao/update.rs

```diff
@@ -2,6 +2,8 @@ use anchor_lang::prelude::*;
 
 use crate::state::*;
 
+// I think we should make these no longer static to allow for futarchy as a service.
+// maybe too much to shove into this?
```

Comment:
Yeah I'm not sure I want to start addressing futarchy-as-a-service in this proposal. It's not scoped out and I think would just lead to me doing whatever random ideas people have.